src/js/actions/ui.js

```diff
@@ -154,10 +158,12 @@ define(function (require, exports) {
                 };
 
                 this.dispatch(events.ui.TRANSFORM_UPDATED, payload);
+            }).then(function () {
+                this.transfer(tools.resetBorderPolicies);
```

Comment:
This also can go in the previous handler, and you probably also want to return the promise.